repo/db/cafe_sessions.go

```diff
@@ -92,17 +94,22 @@ func (c *CafeSessionDB) handleQuery(stm string) []repo.CafeSession {
 			continue
 		}
 
-		var rcafe repo.Cafe
+		var rcafe *pb.Cafe
 		if err := json.Unmarshal(cafe, &rcafe); err != nil {
 			log.Errorf("error unmarshaling cafe: %s", err)
 			continue
 		}
 
-		ret = append(ret, repo.CafeSession{
+		timestamp := timestamp.Timestamp{
+			Seconds: expiryInt,
+			Nanos:   0,
```

Comment:
And then `Nanos` can be used here.

Reply:
I'm a little confused by this situation. I made the change where we're storing UnixNanos in the db... That means that `expiryInt` is nanoseconds from the epoch here. We're creating a pb `Timestamp`, where (my understanding is that) `Seconds` is seconds from epoch and `Nanos` are the remainder nanoseconds beyond that number of seconds. Is that true? If so, we need to do the math to get the number of seconds and remainder nanoseconds?

Reply:
Right yeah, you'd have to do the math... but go's `time.Unix` method does that for you if you only supply nanos... so maybe easiest to just use that, then convert that to proto

Reply:
I don't think that's true about `time.Unix`... The docs:
```Unix returns the local Time corresponding to the given Unix time, sec seconds and nsec nanoseconds since January 1, 1970 UTC. It is valid to pass nsec outside the range [0, 999999999].```

Reply:
Ahhh IS valid to pass nano outside that range. Read that wrong.

Reply:
Yeah, it's like, if nanos appear to _not be a remainer_, use them as such